content/cybersecurity/concepts/cyber-resilience/cyber-resilience.md

```diff
@@ -0,0 +1,33 @@
+---
+Title: 'Cyber Resilience'
+Description: 'Cyber Resilience is the ability of an organization to withstand and recover from cyber incidents.'
+Subjects:
+  - 'Cybersecurity'
+Tags:
+  - 'Cybersecurity'
+  - 'Infrastructure'
+CatalogContent:
+  - 'paths/cyber-resilience-risk-management'
+  - 'paths/fundamentals-of-cybersecurity'
+---
+
+**Cyber Resilience** is the ability of an organization to withstand and recover from cyber incidents. It describes the ability to continue operating despite experiencing challenging cyber events such as ransomware, Distributed Denial of Service (DDoS), social engineering, and insider threats.
+
+## Cyber Resilience In-Use
+
+Becoming cyber resilient is essential to preventing business disruptions that may result from a cybersecurity incident. In order to become cyber resilient, an organization must first consider its current state security posture. Most importantly, the so-called crown jewels (i.e. critical business/service infrastructure) must be identified, prioritized, and protected. Actions such as application control and implementing the principle of least privilege can lessen risk and increase resilience.
+
+## Ways to Achieve Cyber Resilience
+
+In Cybersecurity, some concepts that go into achieving Cyber Resiliency are:
+- **Redundancy:** Refers to having alternate ways to maintain the availability of a resource.
+- **Replication:** Refers to having multiple copies of the same data available in multiple locations.
+- **Non-Persistence:** Refers to the ability to maintain a system’s integrity despite multiple attempts of changes by the users or attackers. 
```

Comment:
```suggestion
- Non-Persistence: Refers to the ability to maintain a system’s integrity despite multiple attempts of changes by the users or attackers. 
```

Reply:
I don't think this actually gets at the core of non-persistence. I understand that this strategy is about limiting the availability of access, or data, etc. I don't think that idea is represented clearly in this definition.